docs/cli-reference/index.md

```diff
@@ -0,0 +1,10 @@
+# CLI Reference
```

Comment:
```suggestion
# Command Line Reference
```
I don't think we need to use the short form here?

Reply:
Valid point. Thanks!